CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+# Contributing
+
+Thanks for your interest in making InsightConnect a better security orchestration tool.
+Before you get started, review our [Code of Conduct].
+
+There are multiple ways to help beyond just writing code:
+
+- [Submit bugs and feature requests] with detailed information about your issue or idea.
+- [Help fellow users with open issues] or [help fellow committers test recent pull requests].
+- [Report a security vulnerability in InsightConnect itself or its plugins] to Rapid7.
```

Comment:
The scope of this repository is only plugins - I'm not sure I would mention the InsightConnect product

Reply:
Done